.travis.yml

```diff
@@ -2,19 +2,19 @@ sudo: false
 dist: trusty
 language: python
 python:
-    - 3.5         # currently Travis trusty default Python3 is 3.5.2
-    #- 3.6.1
-    #- "nightly"  # currently points to 3.7-dev (3.7.0a0)
+    - "3.5.0"
+    - "3.6.1"
+    - "nightly"  # currently points to 3.7-dev
 install:
-    - pip install flake8 pytest
+    - pip install flake8, pytest
```

Comment:
I just backed `.travis.yml` out of the PR...  There are only 5 files still in the PR and their filename are all of the form `test_*.py`.